docs/source/debugging.rst

```diff
@@ -2,18 +2,18 @@ Debugging
 =========
 The following are flags that make debugging much easier.
 
-Fast dev run
-------------
+Unit test
+---------
 This flag runs a "unit test" by running 1 training batch and 1 validation batch.
```

Comment:
does it also run 1 test batch now?

Reply:
yes. It will run for a single test batch.